### PR TITLE
Include functionalities into signup module.

### PR DESCRIPTION
**Pull request by jparrax**
New validations were included into js/validations to do client-side validations
File signupValidation was created to do the server-side validation of user name and email.
Signup view file was changed to include the new functionalities.
I deleted terms from Footer because at this moment it is not necessary.

### CODE REVIEW

File: signupValidation.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    // Query 
+    $sql = "SELECT login
+                from users
+                where login = " . $login ;
+    
+    $result = $conn->query($sql);
+    // Result in case to find something
+    if ($result->num_rows > 0) {
+        echo "This user name is being used";
+    }
```
Comment (AntonRozhanskii) [inline signupValidation.php]: Rewrite this code in this way:
https://www.w3schools.com/php/php_mysql_prepared_statements.asp

File: signupValidation.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    if ($conn->connect_error) {
+        die("Connection failed: " . $conn->connect_error);
+    }
+    // Query 
+    $sql = "SELECT email FROM users WHERE email = \"" .$email. "\"" ;
+    $result = $conn->query($sql);
+    // Result in case to find something
+    if ($result->num_rows > 0) {
+        echo "This email is being used";
+    }
```
Comment (AntonRozhanskii) [inline signupValidation.php]: Rewrite this code in this way:
https://www.w3schools.com/php/php_mysql_prepared_statements.asp

**Review by AntonRozhanskii — CHANGES_REQUESTED** [review]
Rewrite sql query code

File: js/validations/signup.js [hunk 3]
```diff
@@ -28,13 +49,26 @@ function checkPasswordLength() {
...
     } else {
         passInput.classList.remove("border-danger");
         passWarning.innerText = "";
     }
 }
 
+function checkConfirmPassword(){
+    var password = document.getElementById('password');
+    var confirmPassword = document.getElementById('confirm-password'); 
+    var confirmPassWarning = document.getElementById('confirmPassWarning');
```
Comment (AntonRozhanskii) [inline js/validations/signup.js]: First, we already use let.
Second, here is difference between them https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var

**Review by AntonRozhanskii — CHANGES_REQUESTED** [review]
Fix

**Comment by jparrax** [comment]
I will start to do it now, I will include the missing parts for the module. Enjoy, the world is crashing!

**Pull request closed without merging**